src/bpfprogram.cpp

```diff
@@ -243,7 +232,7 @@ void BpfProgram::relocateFuncInfos()
         btfstr + info_sec->sec_name_off);
     if (text_offset_ > 0 && name == ".text")
       text_funcinfo_sec = info_sec;
-    else if (name == name_)
+    else if (name == bpf_program__section_name(bpf_prog_))
```

Comment:
should we just store the section name as a member variable? I see `bpf_program__section_name` called from multiple methods.

Reply:
It's stored as a member of `bpf_prog_` so the only thing this would get us is few chars on a shorter accessor :-) I don't think it's worth it.

Reply:
kk, no worries